mlxfwops/lib/components/fs_comps_ops.h

```diff
@@ -1,34 +1,15 @@
 /*
- * Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * Copyright (c) 2013-2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
  *
```

Comment:
prease restore dual license open-source copyright you overwrote with proprietary one